include/arch/riscv/arch/32/mode/hardware.h

```diff
@@ -62,46 +62,3 @@
 
 #define LOAD  lw
 #define STORE sw
-
```

Comment:
this commit seems to shuffle code around with a new convention that we don't currently follow in the kernel?

Reply:
Putting C code in hardware.h does not scale too well in my dev branches, especially when this starts pulling in more include dependencies. Thus I'm moving this to machine.h, there almost everything can be included without problems. Also, the assembler never sees this file, so it does no need defines.

Reply:
Did you consider putting new C code in a different file,  and leaving the existing code as is?

Reply:
Yes, but since the ARM port also has the special register access wrapper code in `machine.h` also, why not follow this on RISC-V also?

Reply:
if it's following an existing convention, the change makes sense.

Reply:
But if it does not like I am trying to see the changes made by this person but I can not see any changes made by him/her yet so now what should take this as?